config/defines.inc.php

```diff
@@ -45,7 +45,7 @@
 }
 
 if (!defined('_PS_DEBUG_PROFILING_')) {
-    define('_PS_DEBUG_PROFILING_', false);
+    define('_PS_DEBUG_PROFILING_', true);
```

Comment:
```suggestion
    define('_PS_DEBUG_PROFILING_', false);
```

Reply:
Yep, I guess this was changed for debugging This should not me merged!!